kuma/settings/common.py

```diff
@@ -527,6 +530,7 @@ def enable_candidate_languages():
 
     # other
     'kuma.humans',
+    'redirect_urls',
```

Comment:
Is this needed? I would think that the `kuma.redirects` app added above should be sufficient.